django/template/defaultfilters.py

```diff
@@ -871,7 +875,9 @@ def filesizeformat(bytes):
     else:
         value = ugettext("%s PB") % filesize_number_format(bytes / PB)
 
-    return avoid_wrapping(value)
+    if negative:
+        value = "&minus;%s" % value
```

Comment:
Is there a reason you are using`&minus;` instead of `-`? It won't work in non-HTML templates.